src/components/Aside/Aside.tsx

```diff
@@ -0,0 +1,28 @@
+import { ReactElement } from "react";
+import classnames from "classnames";
+
+import "./_aside.scss";
+
+type Props = {
+  children: ReactElement | ReactElement[];
+  width?: "wide" | "narrow";
+  pinned?: boolean;
+};
+
+export default function Aside({
```

Comment:
Should this be part of `react-components` as it's part of the vanilla app layout? 🤷‍♂️

Reply:
We only need to wrap it here to support Framer animations.